source/MaterialXView/Editor.cpp

```diff
@@ -164,38 +287,20 @@ void PropertyEditor::addItemToForm(const mx::UIPropertyItem& item, const std::st
     // Float widget
     else if (value->isA<float>())
     {
-        new ng::Label(container, label);
-        float v = value->asA<float>();
-        auto floatVar = new ng::FloatBox<float>(container, v);
-        floatVar->setFixedSize(ng::Vector2i(100, 20));
-        floatVar->setSpinnable(editable);
-        floatVar->setEditable(editable);
-        floatVar->setFontSize(15);
-        if (min)
-            floatVar->setMinValue(min->asA<float>());
-        if (max)
-            floatVar->setMaxValue(max->asA<float>());
-        floatVar->setCallback([path, viewer](float v)
-        {
-            MaterialPtr material = viewer->getSelectedMaterial();
-            if (material)
-            {
-                mx::ShaderPort* uniform = material ? material->findUniform(path) : nullptr;
-                if (uniform)
-                {
-                    material->getShader()->bind();
-                    material->getShader()->setUniform(uniform->getName(), v);
-                }                
-            }
-        });
+        ng::Widget* threeColumns = new ng::Widget(container);
```

Comment:
I think all the time I have is to look at single float widgets. It's hard to avoid duplication so leaving the vec2, vec3, vec4 as spinners for now.